expression/builtin_info_vec.go

```diff
@@ -44,11 +44,22 @@ func (b *builtinDatabaseSig) vecEvalString(input *chunk.Chunk, result *chunk.Col
 }
 
 func (b *builtinConnectionIDSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinConnectionIDSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	data := b.ctx.GetSessionVars()
+	if data == nil {
+		return errors.Errorf("Missing session variable when evalue builtin")
```

Comment:
1. `when evalue builtin` ==> in \`builtinConnectionIDSig.vecEvalInt\`
2. also refine the error message in builtinConnectionIDSig.evalInt